### ISSUE TITLE
Author in search index (from test)

### ISSUE THREAD

**Issue by roopeux** [buda-base/public-digital-library#940]
Add authors as searchable fields, like `authorName_bo_x_ewts`.  The field will not be shown in the UI, only used for matching.

**Comment by eroux** [comment]
Yes, let's make things a bit more elaborate: when there is only one author (in the author field, which is the person id), the UI could display the (highlighted) matches in the `authorName_*` fields with the link to the person record instead of the usual prefLabel of the person record fetched after the query

**Comment by eroux** [comment]
(But no change if there's no match in the `authorName_*` fields

**Comment by roopeux** [comment]
Yes, I am generally against links within links, because it makes users hesitate.  Since the whole search result div should be a link, it is actually better to not link from authors' names.

**Comment by eroux** [comment]
ah ok, that's quite a big change (even from the old website), can you open a dedicated issue for that?

**Comment by roopeux** [comment]
Yes, we can check that later.  Let's see how this style works in the test.  My point was in the context of adding links from names.
